### DISCUSSION TITLE
Add detection rule for Google Cloud API keys

### DISCUSSION THREAD

**Discussion by roshaaaan** [aquasecurity/trivy#7676] [Ideas]
### Description

I propose adding a rule to detect Google Cloud API keys in the secret scanner. These keys usually start with "AIza" followed by 35 alphanumeric characters, dashes, or underscores. This would help enhance security by identifying potential leaks of sensitive Google Cloud credentials. (I can raise a PR for the change)

### Target

Container Image

### Scanner

Secret

**Comment by DmitriyLewen** [comment]
Hello @roshaaaan 
Is there any documentation about this key format?

> I can raise a PR for the change

It will be great! We are always happy to welcome new contributors.
